packages/blockchain-wallet-v4-frontend/src/components/Chart/components/Tooltip/styles.ts

```diff
@@ -0,0 +1,8 @@
+import styled, { css } from 'styled-components'
+
+export const TooltipCard = styled.div`
+  ${({ theme }) => css`
+    border-radius: 4px;
```

Comment:
magic number

Reply:
is there a constant or an attribute in the theme that I can use to set the card radius here?
like a `border-radius: ${theme.shape.borderRadius}`

Reply:
there will be! but not now